src/modules/Users.pm

```diff
@@ -3996,21 +3973,8 @@ sub WriteCustomSets {
 ##------------------------------------
 # Writes settings to /etc/defaults/useradd
 sub WriteLoginDefaults {
-
-    my $self	= shift;
-    my $ret 	= 1;
-
-    foreach my $key (keys %useradd_defaults) {
-	my $value	= $useradd_defaults{$key};
-	$ret = $ret && SCR->Write (".etc.default.useradd.\"\Q$key\E\"", $value);
-    }
-
-    if ($ret) {
-	SCR->Write (".etc.default.useradd", "force");
-    }
-    y2milestone ("Succesfully written useradd defaults: $ret");
-    y2usernote ("File '/etc/default/useradd' was modified.");
-    return $ret;
+    y2error ("Calling obsolete WriteLoginDefaults");
```

Comment:
is it really called anywhere? As it does not have typedef macro, it cannot be called from yast. Maybe it can be just removed?